projects/npm-tools/packages/npm-scripts/src/jest/mocks/Liferay.js

```diff
@@ -36,6 +67,18 @@ const events = {
 	once: jest.fn(),
 };
 
+const namespace = jest.fn((name) => {
+	global.Liferay[name] = {};
+
+	return global.Liferay[name];
+});
```

Comment:
Where does this come from? I'm not super familiar with this one

Reply:
It's used in here https://github.com/liferay-frontend/liferay-portal/blob/master/modules/apps/portal-search/portal-search-web/test/setup.js#L64-L70